shared/dataflow/codeql/dataflow/TaintTracking.qll

```diff
@@ -0,0 +1,83 @@
+/**
+ * Provides classes for performing local (intra-procedural) and
+ * global (inter-procedural) taint-tracking analyses.
+ */
+
+import DataFlow
+import DataFlowImpl
+import DataFlowParameter
+import TaintTrackingParameter
+
+module TaintFlowMake<
+  DataFlowParameter DataFlowLang, TaintTrackingParameter<DataFlowLang> TaintTrackingLang>
+{
+  private import DataFlowLang
+  private import TaintTrackingLang
+  private import DataFlowMake<DataFlowLang>
+  private import MakeImpl<DataFlowLang>
```

Comment:
Maybe keep the qualifiers here?
```suggestion
  private import DataFlowMake<DataFlowLang> as DataFlow
  private import MakeImpl<DataFlowLang> as DataFlowInternal
```
Then we can also refer to the nodes below as `DataFlow::Node` rather than `DataFlowLang::Node`.

Reply:
Added the qualifiers. Still need to address the `DataFlowLang::Node` issue.

Reply:
Replacing `DataFlowLang::Node` by `DataFlow::Node` gives me an unresolved type error.

Reply:
> Huh, I guess I was too quick with the "LGTM" comment below.

Feel free to retract your LGTM 😄

Reply:
Ah, I guess that does make sense - it's the individual languages that re-export `Node` as `DataFlow::Node` - not the shared data flow lib.

Reply:
So my "LGTM" stands 😄